packages/client/src/workflow-client.ts

```diff
@@ -138,30 +138,49 @@ export interface WorkflowHandle<T extends Workflow = Workflow> extends BaseWorkf
   ): Promise<Ret>;
 
   /**
-   * Start an Update and receive a handle to the Update.
-   * The Update validator (if present) is run before the handle is returned.
+   * Start an Update and receive a handle to the Update. The Update validator (if present) is run
+   * before the handle is returned.
    *
    * @experimental Update is an experimental feature.
    *
    * @throws {@link WorkflowUpdateFailedError} if Update validation fails.
    *
    * @param def an Update definition as returned from {@link defineUpdate}
-   * @param options Update arguments
+   * @param options update arguments, and update lifecycle stage to wait for
+   *
+   * Currently, startUpdate always waits until a worker is accepting tasks for the workflow and the
+   * update is accepted or rejected, and the options object must be at least
+   * ```ts
+   * {
+   *   waitForStage: temporal.api.enums.v1.UpdateWorkflowExecutionLifecycleStage.UPDATE_WORKFLOW_EXECUTION_LIFECYCLE_STAGE_ACCEPTED
```

Comment:
TODO: offer a more ergonomic way to import this value

Reply:
We need to fix this before we merge this PR. Fixing this later would be API breaking.

Look at `ScheduleOverlapPolicy` in schedule-client.ts/schedule-helpers.ts/schedule-types.ts, or `VersioningIntent` in `versioning-intent-enum.ts`. I'm ok with either approaches.

Reply:
Agreed, this is a blocking TODO (I should have marked the PR as draft).